sql-ballerina/tests/complex-query-test.bal

```diff
@@ -83,7 +83,7 @@ function testToJson() {
         BOOLEAN_TYPE: true,
         STRING_TYPE: "Hello"
     };
-    test:assertEquals(retVal, expectedData.cloneWithType(json), "Expected JSON did not match.");
+    test:assertEquals(retVal, checkpanic expectedData.cloneWithType(json), "Expected JSON did not match.");
```

Comment:
Shall we add `check` and returns the error from the test case. I think it also fails the test case.

Reply:
Fixed in 6371723